pkg/vz/vz_driver_darwin.go

```diff
@@ -0,0 +1,62 @@
+package vz
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/lima-vm/lima/pkg/store/filenames"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Code-Hex/vz/v3"
+	"github.com/lima-vm/lima/pkg/driver"
+	"github.com/lima-vm/lima/pkg/limayaml"
+)
+
+type LimaVzDriver struct {
+	*driver.BaseDriver
+
+	machine *vz.VirtualMachine
+}
+
+func New(driver *driver.BaseDriver) *LimaVzDriver {
+	return &LimaVzDriver{
+		BaseDriver: driver,
+	}
+}
+
+func (l *LimaVzDriver) Validate() error {
+	if *l.Yaml.MountType == limayaml.NINEP {
+		return fmt.Errorf("field `mountType` must be %q or %q for VZ driver , got %q", limayaml.REVSSHFS, limayaml.VIRTIOFS, *l.Yaml.MountType)
+	}
+	return nil
```

Comment:
Warnings should be printed for unsupported YAML propertries: https://github.com/containerd/nerdctl/blob/c1a4159035b77a4a739a503b98139a17ac3440b8/pkg/composer/serviceparser/serviceparser.go#L38

Can be another PR though

Reply:
Added errors for not supported one like legacyBios. As vm will not even start with this as true. 
For other things will take it up in a follow-up PR like networks, mount as 9p etc.